examples/python/azure/README.md

```diff
@@ -82,7 +82,7 @@ The above command will create a folder called `cdktf.out` that contains all Terr
 Run Terraform commands
 
 ```bash
-cd cdktf.out
+cd cdktf.out/stacks/python-azure
```

Comment:
Why not use the cdktf commands here as well?

Reply:
They're mentioned below, but I'm going to remove the "Terraform way" altogether anyway :)